server/services/store/sqlstore/workspaces.go

```diff
@@ -144,3 +145,62 @@ func (s *SQLStore) GetWorkspaceCount() (int64, error) {
 	}
 	return count, nil
 }
+
+func (s *SQLStore) GetUserWorkspaces(userID string) ([]model.UserWorkspace, error) {
+	var query sq.SelectBuilder
+
+	if s.dbType == mysqlDBType {
+		query = s.getQueryBuilder().
+			Select("Channels.ID", "Channels.DisplayName", "COUNT(focalboard_blocks.id)").
+			From("focalboard_blocks").
+			Join("ChannelMembers ON focalboard_blocks.workspace_id = ChannelMembers.ChannelId").
+			Join("Channels ON ChannelMembers.ChannelId = Channels.Id").
+			Where("ChannelMembers.UserId = ?", userID).
```

Comment:
I prefer to use here the `sq.Eq{"ChannelMembers.UserId": userID}` approach, is more explicit.

Reply:
same in the next line, and in the postgres queries.

Reply:
Done.